packages/mermaid/src/diagrams/gantt/ganttDb.spec.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @cspell/spellchecker
```

Comment:
```suggestion
```

Please remove and just add `ts-nocheck` to the ignored words list (we probably use `ts-nocheck` a lot).